### PR TITLE
Review of the design doc before initial publication.

### PR DESCRIPTION
**Pull request by kantp**
Also, given more thought about how this will work with Praos.

Also, giving the whole document an overhaul, pruning obsolete things, a couple of grammar edits, etc.

### CODE REVIEW

File: shelley/design-spec/delegation_design_spec.tex [hunk 1]
```diff
@@ -74,11 +74,12 @@ \section*{List of Contributors}
...
 \label{purpose}
 
-Delegation will allow holders of ada to transfer their rights to
-participate in the proof of stake (\emph{PoS}) protocol to \emph{stake
-pools}. Stake pools are run by \emph{stake pool operators} (also called
-\emph{pool leaders}), and a person delegating to a stake pool is called
-\emph{delegator}, \emph{member}, or \emph{participant} of a stake pool.
+Delegation will allow holders of ada to transfer their rights to participate in
+the proof of stake (\emph{PoS}) protocol to \emph{stake pools}. Stake pools are
+run by \emph{stake pool operators} (sometimes also called \emph{pool leaders},
```
Comment (dnadales) [inline shelley/design-spec/delegation_design_spec.tex]: Just curious: I'm just wondering in which context the term "pool leader" makes sense. What are they leading?

File: shelley/design-spec/delegation_design_spec.tex [hunk 2]
```diff
@@ -248,9 +240,15 @@ \subsubsection{Handle Inactive Stake Pools}
 \label{handle-inactive-stake-pools}
 
 We anticipate that a stake pool operator can cease to operate -- whether
-they lost their keys, lost interest, died, etc. We want to minimise the
```
Comment (dnadales) [inline shelley/design-spec/delegation_design_spec.tex]: As of this PR, stake pools operators are not allowed to die anymore :P I understand the change though ...

File: shelley/design-spec/delegation_design_spec.tex [hunk 3]
```diff
@@ -1617,39 +1613,76 @@ \subsubsection{Proposal}
...
 
-\subsubsection{Plan}
-\label{plan}
+We do this soft transition as a derisking strategy, so that we can intervene in
+case we observe any difficulties in the decentralised system. But we do not want
+this to have an effect on the rewards that pools get. Operational difficulties
+of the overall system that cause us to slow down the transition should not
+reduce the rewards that individual pools get.
+
+In order to minimise the influence of the transition to pool rewards, we have to
```
Comment (dnadales) [inline shelley/design-spec/delegation_design_spec.tex]:
```suggestion
In order to minimise the influence of the transition on pool rewards, we have to
```
?

File: shelley/design-spec/delegation_design_spec.tex (hunk 2 above)
Comment (kantp) [reply shelley/design-spec/delegation_design_spec.tex]: I was told it was a bit morbid :)

**Pull request merged**